lib/kender/commands/cypress_rails.rb

```diff
@@ -0,0 +1,16 @@
+module Kender
+  class CypressRails < Command
+
+    def available?
+      in_gemfile?('cypress-rails')
+    end
+
+    # We check if the rake taks is defined before using it.
+    # Kender might be in some projects which use cypress-rails but
+    # do not have this rake task.
+    def command
+      'if rake -T | grep "rake cypress_extended:enhanced_run"; then rake cypress_extended:enhanced_run; fi'
```

Comment:
With the check done in `available?`, now just do:
```suggestion
      'bundle exec rake cypress_extended:enhanced_run'
```